module/data/item/templates/ability.mjs

```diff
@@ -47,6 +48,14 @@ export default class AbilityTemplate extends ActionTemplate.mixin(
                 integer: true,
                 label: "ED.Item.Ability.rank"
             } ),
+            combatAbilityType: new fields.StringField( {
```

Comment:
How does this come into play in this branch? What do we need it for?

Reply:
I added the combatTypeto assign the flavor htmls to it but i see that the combat flavor types are biting with the rolltypes...

We will need the combatTypes for differentiation of the abilities, but I removed the falvorTemplate, the differences have to be made inside the attack template based on the combatType

Reply:
Combat Type is necessary for the ability to recognize which type of weapon is required for the usage
but you are right it has not right to be in here, I commented it out